mm2src/lp_ordermatch.rs

```diff
@@ -2619,6 +2621,7 @@ struct OrdermatchContext {
     pending_maker_reserved: AsyncMutex<HashMap<Uuid, Vec<MakerReserved>>>,
     #[cfg(target_arch = "wasm32")]
     ordermatch_db: ConstructibleDb<OrdermatchDb>,
+    pub balance_loops: AsyncMutex<HashSet<String>>,
```

Comment:
It doesn't have to be `AsyncMutex`. You can use `parking_lot::Mutex` without async overhead.